cedar-policy-cli/src/lib.rs

```diff
@@ -71,7 +71,7 @@ pub struct ValidateArgs {
 pub struct CheckParseArgs {
     /// File containing the policy set
     #[clap(short, long = "policies", value_name = "FILE")]
-    pub policies_file: String,
+    pub policies_file: Option<String>,
```

Comment:
A comment about what `None` implies would be helpful here.  (The comment gets incorporated into the `--help` by `clap`)